app/controllers/admin/invoices_controller.rb

```diff
@@ -1,4 +1,17 @@
 class Admin::InvoicesController < ApplicationController
   def index
+    @invoices = Invoice.all
+  end
+
+  def show
+    @invoice = Invoice.find(params[:id])
+    @invoice_items = InvoiceItem.find_items(@invoice)
+  end
+
+  def update
+    invoice = Invoice.find(params[:id])
+    invoice.update!(status: params[:status])
```

Comment:
would we potentially want to consider putting :status into strong params in a private method at the bottom?